offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -236,7 +236,8 @@ class CreatePersonView extends VerticalLayout {
     private void refreshAddressAdditions() {
         ListDataProvider<Affiliation> dataProvider = this.addressAdditionComboBox.dataProvider as ListDataProvider<Affiliation>
         dataProvider.clearFilters()
-        dataProvider.addFilterByValue({it.organisation},createCustomerViewModel.affiliation.organisation)
+        dataProvider.addFilterByValue({it.organisation },
+                createCustomerViewModel.affiliation?.organisation)
```

Comment:
Nice catch! What happens here in case the value of `affiliation` is null? Are the addressAddition values cleared?